### PR TITLE
switch to Thym 0.1 stable site now that it's released;
JBIDE-19045 support Tern & Angular 0.7 while keeping 0.6 in the target platforms >> jbosstools-4.2.x

### PR DESCRIPTION
**Pull request by nickboldt**

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -80,8 +80,12 @@
 		<jbosstools-browsersim-site>${jbosstools-site-baseurl}/jbosstools-browsersim_${jbosstools-site-stream}/${jbosstools-site-suffix}</jbosstools-browsersim-site>
 		<jbosstools-webservices-site>${jbosstools-site-baseurl}/jbosstools-webservices_${jbosstools-site-stream}/${jbosstools-site-suffix}</jbosstools-webservices-site>
 
+		<!-- TODO: when changing these stable or integration URLs, be sure to ALSO change the URLs in download.jboss.org/jbosstools/builds/staging/_composite_/core/* or else composite builds will fail and no aggregates will happen in Jenkins! -->
 		<!-- JBIDE-18869: use specific releases composited together from latest at eclipse, mirroned onto dl.jb.org -->
-		<eclipse-thym-site>http://downloads.jboss.org/jbosstools/updates/integration/luna/core/thym/</eclipse-thym-site>
+		<eclipse-thym-site>http://downloads.jboss.org/jbosstools/updates/stable/luna/core/thym/</eclipse-thym-site>
```
Comment (maxandersen) [inline parent/pom.xml]: do we consider 0.1 the *stable* one for JBoss Tools ? I would have expected it to be outside of stable until we actually release it as part of JBoss Tools ?

Comment (dgolovin) [inline parent/pom.xml]: I guess Nick's changes are about searching stable location so there is no need to change parent/pom.xml when thym/tern version changed.

Comment (nickboldt) [inline parent/pom.xml]: Yes, because for 8.1.x maintenance it'd be great to not have to change the parent pom to pull in a new version of Thym. :D

**Comment by dgolovin** [comment]
Closing, not required anymore, because tern is aggregated into JST module p2 repo and then to JBT aggregated p2repo.

**Comment by nickboldt** [comment]
Uh, this was for Thym, not Tern.

**Pull request closed without merging**